data/hai ships.txt

```diff
@@ -411,7 +411,7 @@ ship "Solifuge"
 	explode "large explosion" 50
 	explode "huge explosion" 20
 	"final explode" "final explosion large"
-	description `Impressed by the Alpha's "Giftbringer", the Unfettered endeavoured to build their own fighter carrier to take more advantage of their supply of Jump Drives.`
+	description `Impressed after seeing their green allies' carrier, "Giftbringer", the Unfettered decided it was time they expanded their fleet. They built the Solifuge carrier to take better advantage of their Jump Drive supply. A capable warship by itself, the Solifuge also carries with it a small fleet of its own, without needing to use extra drives.`
```

Comment:
`green allies'`
Don't.

`Impressed by the Alpha's "Giftbringer," [...]` Then the rest. The first part was better originally (excluding the fact that the comma should be inside the quotes.)

Also, "jump drive" is generally all lowercase.